src/sds_data_model/constants.py

```diff
@@ -52,3 +52,54 @@ def _get_bboxes(
 
 # A tuple of BoundingBox for BNG 100km grid squares.
 BBOXES = _get_bboxes()
+
+TITLE_XPATH = [
+    "gmd:identificationInfo",
+    "gmd:MD_DataIdentification",
+    "gmd:citation",
+    "gmd:CI_Citation",
+    "gmd:title",
+    "gco:CharacterString",
+]
+
+ABSTRACT_XPATH = [
+    "gmd:MD_Metadata",
+    "gmd:identificationInfo",
+    "gmd:MD_DataIdentification",
+    "gmd:abstract",
+    "gco:CharacterString"
+]
+
+LINEAGE_XPATH =[
+    "gmd:MD_Metadata",
+    "gmd:dataQualityInfo",
+    "gmd:DQ_DataQuality",
+    "gmd:scope",
+    "gmd:DQ_Scope",
+    "gmd:level",
+    "gmd:MD_ScopeCode"
```

Comment:
This is the the data quality scope, for lineage it would be:

```suggestion
    "gmd:lineage",
    "gmd:LI_Lineage",
    "gmd:statement",
    "gco:CharacterString",
```